recipes/c-ares/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
this two meta lines can be removed as well

Reply:
I prefer to keep the encoding there

Reply:
Python 3 is UTF-8 by default
Also I think we should have things like these uniform across all recipes.
At least in Bincrafters we decided a while ago to remove these lines

Reply:
I totally agree with you. This is something we should move to the conan-center hook. Would you mind opening the corresponding issue (probably in this repo to agree about that, and then in the hooks repo to do the work).

Meanwhile, these recipes should work with py2 too and, although this one doesn't have any non-ascii chars in it, someone could use to copy/paste. I would vote to add the encoding line, but let's wait for an agreement.

Thanks a lot for your review and you interest.

Reply:
I've created issues for this small adjustments here: https://github.com/conan-io/hooks/issues